docs/guides/getting-started-java-spring.md

```diff
@@ -0,0 +1,255 @@
+3. Under **Project Metadata**, configure the following:
+   1. **Group**: `io.camunda.demo`
+   2. **Artifact**: `process_payments`
+   3. **Name**: `Process payments`
+   4. **Description**: `Process payments with Camunda`
+   5. **Package name**: `io.camunda.demo.process_payments`
+   6. **Packaging**: `Jar`
+   7. **Java**: Select the Java version you have installed.
+   8. For this tutorial, we will not install any dependencies.
+4. Click **Generate**.
+5. Download the project, click into the `.zip` file, and add it to your desired location.
+6. Open this project in your preferred code editor.
+7. Run `mvn spring-boot:run` in your terminal to confirm your Spring project builds.
+8. (Optional) Run `git init` if you'd like to commit milestones along the way, and add a `.gitignore` file with `target/` to ignore build artifacts.
+
+## Step 3: Create a new BPMN diagram
+
+Next, we'll create a BPMN diagram to represent the transaction model shown at the beginning of this guide:
+
+1. Open Desktop Modeler.
+2. Click **Create a new diagram** in Camunda 8, and name your diagram `Process payments` with an id of `process-payments`.
+3. Add a start event, and name it `Payment request received`.
+4. Add a task named `Prepare transaction`. Click the wrench-shaped icon to change the type of task to a script task, and configure the following properties:
+   1. **Implementation**: `FEEL expression`
+   2. **Script/Result variable**: `totalWithTax`
+   3. **Script/FEEL expression**: `total * 1.1` (this represents the tax applied to the transaction.)
+5. Add a task named `Charge credit card`.
+6. Click on the task and click the wrench-shaped icon to change the type of task to a service task. In the properties panel, change the **Task definition/Type** to `charge-credit-card`.
+7. Add an end event named `Payment executed`.
+8. Save this BPMN file to your Spring project in `src/main/resources`, and name it `process-payments.bpmn`.
+
+## Step 4: Deploy your process
+
+To deploy your process, take the following steps:
+
+1. Open Desktop Modeler and click the rocket icon in the bottom left corner.
+2. Change the **Deployment name** to `process-payments`, and ensure the **Target** is `Camunda 8 Self-Managed`.
+3. Change the **Cluster endpoint** to `http://localhost:26500/`, with no authentication.
+4. Click **Deploy**.
+
+When you open Operate, you should now note the process deployed to your local Self-Managed setup.
+
+## Step 5: Run your process from Modeler
+
+To run your process, take the following steps:
+
+1. From Desktop Modeler, click the "play" icon (next to the rocket icon to deploy) in the bottom left corner.
+2. In **Variables**, insert the JSON object `{"total": 100}`.
+3. Click **Start**.
+
+From Operate, you should now notice a process instance running. You'll notice the process has stopped at **Charge credit card**, because we'll need to configure a job worker.
```

Comment:
```suggestion
From Operate, you should now notice a process instance running. You'll notice the process instance is waiting at **Charge credit card**, because we'll need to configure a job worker.
```

precision